zap/src/main/java/org/zaproxy/zap/extension/api/NodeJSAPIGenerator.java

```diff
@@ -136,7 +164,7 @@ private void generateNodeJSElement(
                                     name ->
                                             "'"
                                                     + name
-                                                    + "': "
+                                                    + "': args."
```

Comment:
args will be concatenated with each parameter, since it will be now passed in an args object

Reply:
I meant add the generate code of a function to the pull request.